CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,19 @@
 # Changelog 1.11
 
+## 1.11.2 - 2023-05-15
+
+### Changed
+
+- Update Open vSwitch to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Fix IPsec e2e flaky. ([#4774](https://github.com/antrea-io/antrea/pull/4774), [@xliuxu])
+- Fix race condition between stale controller and resourceimport reconcilers. ([#4853](https://github.com/antrea-io/antrea/pull/4853), [@Dyanngg])
+- Fix FQDN upper-case issue. ([#4934](https://github.com/antrea-io/antrea/pull/4934), [@GraysonWu])
+- Run modules that rely on Services after AntreaProxy is ready. ([#4946](https://github.com/antrea-io/antrea/pull/4946), [@tnqn])
```

Comment:
Please check PR4790 to update the PR's description, and put them into right section. It should be in `Fixed` section instead of `Changed`.